app/jobs/hackathons/archive_websites_job.rb

```diff
@@ -10,7 +10,6 @@ def perform
   private
 
   def upcoming_or_recent_hackathons
-    # Hackathons which are yet to happen or finished in the past month
```

Comment:
Why delete this, isn't it helpful?

Reply:
I think it's kinda redundant since it's only one line that basically says the same thing. If you think otherwise I'd much rather find a way to make the code clearer than adding a separate comment.